### PR TITLE
[w5.4][T14-A3]Ma Junting

### PR DESCRIPTION
**Pull request by majunting**

### CODE REVIEW

**Comment by LuLechuan** [comment]
Perhaps make the "Group" optional for each person. Meaning that for a new person added, he/she can be with or without a group.

File: doc/UserGuide.md [hunk 1]
```diff
@@ -66,6 +66,16 @@ Examples:
 * `find Betsy Tim John`<br>
   Returns Any person having names `Betsy`, `Tim`, or `John`
 
+*** Finding all persons within group: `findgroup`
```
Comment (jaime-lanister) [inline doc/UserGuide.md]: Probably better to write as findGroup?

File: src/seedu/addressbook/data/person/Person.java [hunk 2]
```diff
@@ -50,6 +52,9 @@ public Email getEmail() {
     }
 
     @Override
```
Comment (jaime-lanister) [inline src/seedu/addressbook/data/person/Person.java]: Why do you need to override here?

**Comment by ang-zy** [comment]
Your tests fail as well, which is worrying. Please make sure they pass.

**Pull request closed without merging**